themes/light/templates/vue-components/file-browser.mako

```diff
@@ -1,11 +1,11 @@
 <script type="text/x-template" id="file-browser-template">
     <div>
-        <input v-model="currentPath" :name="name" type="text" id="episodeDir" class="input-sm form-control form-control-inline fileBrowserField"/>
-        <input v-if="showBrowseButton" type="button" :value="'Browse\u2026'" class="btn btn-inline fileBrowserButton"/>
-        <div class="fileBrowserDialog" style="display:hidden"></div>
+        <input v-model="currentPath" ref="locationInput" :name="name" @keyup.enter="browse($event.target.value)" type="text" class="input-sm form-control fileBrowserField" style="width: 70%;" />
```

Comment:
This `@keyup.enter="browse($event.target.value)"` might have made it in by mistake, I'll have to test.